stats/internal/otel/prometheus/exporter.go

```diff
@@ -0,0 +1,425 @@
+package prometheus
```

Comment:
Against which version of the original exporter did we perform our changes? Shall we include markers indicating our customisations? We will definitely need to reapply our customisations after an otel version upgrade, since as it seems, the exporter code is still evolving.

Reply:
@atzoum [here](https://github.com/rudderlabs/rudder-go-kit/pull/13/commits/6fb979231997e1402143487007c654b09f11b643)